### ISSUE TITLE
Space between the profile and the social links

### ISSUE THREAD

**Issue by raisedadead** [freeCodeCamp/freeCodeCamp#37324]
![image](https://user-images.githubusercontent.com/1884376/66879431-a5cc0b80-efdb-11e9-9c78-22769f920fdd.png)

**Comment by Frozenaught** [comment]
Hello, can I work on this please?

**Comment by ojeytonwilliams** [comment]
@Frozenaught, go for it. If it's been labelled 'help wanted', you don't need to ask.

**Comment by dboydgit** [comment]
Which page is this on? I can't find the social links on my local setup.

**Comment by ojeytonwilliams** [comment]
@dboydgit did that bug ever reappear?

**Comment by dboydgit** [comment]
@ojeytonwilliams Yes, when a user adds a new social link to their profile it won't show up without a page refresh. I was going to add a new issue for it.

**Comment by ahmadabdolsaheb** [comment]
> Yes, probably quite a few updates going to the API that are not getting updated in state. I'm a complete react newbie though, so hard for me to figure out what isn't getting updated.

this resolves it #37374
